src/olympia/reviewers/tests/test_utils.py

```diff
@@ -150,7 +149,7 @@ def check_log_count(self, id):
 # it works correctly, so we set ENABLE_ADDON_SIGNING to True and mock the
 # actual signing call.
 @override_settings(ENABLE_ADDON_SIGNING=True)
-@mock.patch('olympia.lib.crypto.signing.call_signing', lambda f: None)
+@patch('olympia.lib.crypto.signing.call_signing', lambda f: None)
```

Comment:
there's a lot of `@patch('olympia.reviewers.utils.sign_file')` changes below - just patch it at the top here too?

Reply:
4a584bb7876abe5b29685639fce960c87e7fb426